hledger-lib/Hledger/Utils/Text.hs

```diff
@@ -169,15 +169,17 @@ stripquotes s = if isSingleQuoted s || isDoubleQuoted s then T.init $ T.tail s e
 
 isSingleQuoted :: Text -> Bool
 isSingleQuoted s =
-  T.length (T.take 2 s) == 2 && T.head s == '\'' && T.last s == '\''
+  T.length s >= 2 && T.head s == '\'' && T.last s == '\''
 
 isDoubleQuoted :: Text -> Bool
 isDoubleQuoted s =
-  T.length (T.take 2 s) == 2 && T.head s == '"' && T.last s == '"'
+  T.length s >= 2 && T.head s == '"' && T.last s == '"'
```

Comment:
I'm a little confused.. was one or both of these causing the reported bug ? I'm not quite seeing why the old code was wrong.

Reply:
No, not wrong. They are completely equivalent, but the old code was much harder for me to parse. Saying that the length is 2 or longer seems much clearer to me than saying that if you take the first two characters, you have exactly two characters.

Reply:
I changed it because I thought that may have been the source, and spent 30 seconds trying to parse it to see if it was. It wasn't, but I thought I should simplify it to make it easier to parse next time.

Reply:
What was the cause of the bug, actually ?

Reply:
`accountNameWithoutPostingType`